packages/tasks/src/model-libraries.ts

```diff
@@ -575,6 +575,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.whisperkit,
 		countDownloads: `path_filename:"model" AND path_extension:"mil" AND _exists_:"path_prefix"`,
 	},
+	"seed-story": {
+		prettyLabel: "SEED-Story",
+		repoName: "SEED-Story",
+		repoUrl: "https://github.com/TencentARC/SEED-Story",
+		filter: false,
+		countDownloads: `path:"pytorch_model.bin" OR path:"tokenizer.model"`,
```

Comment:
Which files from https://huggingface.co/TencentARC/SEED-Story/tree/main are always downloaded when using this model? For example, is [cvlm_llama2_tokenizer/tokenizer.model](https://huggingface.co/TencentARC/SEED-Story/blob/main/cvlm_llama2_tokenizer/tokenizer.model) always downloaded? I'm asking because it's usually best to explicitly define a single file. The problem with defining `pytorch_model.bin` is that there are 4 (?) of them in the repo. If a user downloads the whole repo, it will be counted as 4 downloads which is not accurate.

Reply:
[cvlm_llama2_tokenizer/tokenizer.model](https://huggingface.co/TencentARC/SEED-Story/blob/main/cvlm_llama2_tokenizer/tokenizer.model) is always downloaded. 

The seedx(https://huggingface.co/TencentARC/SEED-Story/tree/main/seedx) and detokenizer-pretrained(https://huggingface.co/TencentARC/SEED-Story/tree/main/detokenizer/detokenizer_pretrained) model is downloaded only when you want to train a new model. 

The seed-story(https://huggingface.co/TencentARC/SEED-Story/tree/main/seed_story) and detokenzier-adapted(https://huggingface.co/TencentARC/SEED-Story/tree/main/detokenizer/detokenizer_george_adapted)  is downloaded when you want to inference.

Do you have any suggestions in that case?

Reply:
Then I think the best os to count on `path:"cvlm_llama2_tokenizer/tokenizer.model"` since it'll always be downloaded when user tries to run the model.